godot_rl/wrappers/ray_wrapper.py

```diff
@@ -45,7 +45,7 @@ def vector_reset(self) -> List[EnvObsType]:
     def vector_step(
         self, actions: List[EnvActionType]
     ) -> Tuple[List[EnvObsType], List[float], List[bool], List[EnvInfoDict]]:
-        actions = np.array(actions)
+        actions = np.array(actions, dtype=object)
```

Comment:
What is the reason for this?

Reply:
I believe this was because of a warning which was fired on every training step.